blueprint/index.md

```diff
@@ -1 +1,210 @@
+1. Open the [Genesys Cloud API Explorer](https://developer.genesys.cloud/developer-tools/#/api-explorer "Opens the Genesys Cloud Developer API Explorer page").
+2. Navigate to **Conversations** > **Messaging**.
+3. Select **GET Get a list of Open messaging integrations**.
+4. Click **Send Request**.
+5. Search for your integration using the `name` field in the Response.
+6. Note the GUID of the integration, which is in the `id` field of your integration. 
+:::primary
+**Note**: You require the GUID of the integration to configure the Ionic front-end framework.
+:::
+
+### Set up Genesys Cloud routing
+Set up routing for the integration to route the messages to your agents or bot.
+
+1. In Genesys Cloud, navigate to **Admin** > **Architect**.
+2. In Architect, create an Inbound Message flow. For more information, see [Add an inbound message flow](https://help.mypurecloud.com/articles/inbound-message-flows/ "Opens the Inbound message flows overview") in the Genesys Cloud Resource Center.
+3. Navigate to **Admin** > **Routing** > **Message Routing**.
+4. Associate the inbound message flow with a message route. Select the open messaging integration as the inbound address to enter the inbound message flow. For more information, see [Message routing](https://help.mypurecloud.com/articles/about-message-routing/ "Opens the Message routing overview article.")
+
+### Configure and run Ionic framework
+Install the Ionic toolkit from the instructions [here](https://ionicframework.com/ "Opens the Ionic framework website").
+1. After installation, populate the environment file ` ./ionic-frontend/src/environments/environment.ts` with the following information:
+   1. Enter the Amazon API Gateway URL for trancriptAPIBaseURL followed by `/transcript/integration/`.
+   2. Enter the Amazon API Gateway URL for notificationAPIBaseURL followed by `/notification/integration/`.
+   3. Specify the GUID of the Genesys Cloud open messaging integration in the `integrations` array. Replace the existing value with the GUID that is already present for `chat` and `offers` objects.
+   4. Provide the `userId` in the format of the email address of the user. 
+For more information about installing and running the project using the Ionic toolkit, see [here](https://github.com/shansrini/ionic-mobile-open-messaging-blueprint/blob/main/ionic-frontend/README.md).
+
+If you have set with Ionic and `npm`, you can navigate to the ionic-frontend folder and execute `ionic serve` in the command prompt.
+
+### How it works?
+
+Once you have set up everything, send a message from the chat widget of the Ionic front-end interface. The chat message is routed to Genesys Cloud Inbound Message flow. The agent responds from the Genesys Cloud agent desktop and Genesys Cloud sends the message to the webhook URL defined in the open messaging integration.
+
+**Agentless setup**
```

Comment:
I think you need another new line after **Agentless setup** so that it doesn't become part of the following sentence.  It should be a header I think.